assets/src/stories-editor/components/right-click-menu/index.js

```diff
@@ -235,7 +240,11 @@ const applyWithDispatch = withDispatch( ( dispatch, props ) => {
 		}
 
 		const isFirstPage = getBlockOrder().indexOf( pageClientId ) === 0;
-		insertBlocks( ensureAllowedBlocksOnPaste( content, pageClientId, isFirstPage ), null, pageClientId );
+		insertBlocks( ensureAllowedBlocksOnPaste( content, pageClientId, isFirstPage ), null, pageClientId ).then( ( { blocks } ) => {
```

Comment:
I don't think [`insertBlocks`](https://developer.wordpress.org/block-editor/data/data-core-block-editor/#insertBlocks) returns a Promise?

Reply:
It returns an action object that is a promise - https://github.com/WordPress/gutenberg/blob/6ecff182908d5cc3f8555879f612f75113629f5e/docs/designers-developers/developers/data/data-core-block-editor.md#insertBlocks

Reply:
This is what I get if console.log `insertBlocks`
```
Promise {<resolved>: {…}}
__proto__: Promise
[[PromiseStatus]]: "resolved"
[[PromiseValue]]: Object
```